### PR TITLE
Develop measure latency

### PR DESCRIPTION
**Pull request by konstantinos-papadopoulos**
Added switches_measurement_latency_interval within nb_active_scalability_multinet.py
The measurement is displayed at the moment within NSTAT log file test. At the moment we will keep it as is.

### CODE REVIEW

File: emulators/multinet/build.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ echo $SCRIPT_DIR
 MULTINET_LOCATION="https://github.com/intracom-telecom-sdn/multinet.git"
 
 if [ ! -d $SCRIPT_DIR"/multinet" ]; then
-    git clone -b master $MULTINET_LOCATION $SCRIPT_DIR"/multinet"
+    git clone -b develop_ip_offset_of_hosts $MULTINET_LOCATION $SCRIPT_DIR"/multinet"
```
Comment (anastop) [inline emulators/multinet/build.sh]: change to master

File: stress_test/nb_active_scalability_multinet.py [hunk 2]
```diff
@@ -214,6 +214,11 @@ def nb_active_scalability_multinet_run(out_json, ctrl_base_dir,
                 multinet_hosts_per_switch, multinet_topology_type,
                 multinet_switch_type, multinet_worker_ip_list,
                 multinet_worker_port_list, multinet_base_dir, 0, 0)
+            """"
```
Comment (anastop) [inline stress_test/nb_active_scalability_multinet.py]: if not needed then remove them (do not commit commented code)

File: stress_test/sample_test_confs/beryllium/beryllium_nb_active_scalability_multinet.json [hunk 3]
```diff
@@ -61,21 +61,21 @@
 "topology_get_switches_handler":"bin/handlers/get_switches",
```
Comment (anastop) [inline stress_test/sample_test_confs/beryllium/beryllium_nb_active_scalability_multinet.json]: the sample test case parameters have to change back to their original values, don't they?

File: stress_test/nb_active_scalability_multinet.py [hunk 4]
```diff
@@ -248,6 +253,12 @@ def nb_active_scalability_multinet_run(out_json, ctrl_base_dir,
                 multinet_rest_server, controller_nb_interface,
                 multinet_base_dir)
 
+            t_start = time.time()
```
Comment (anastop) [inline stress_test/nb_active_scalability_multinet.py]: why was this command for querying the switches added?

File: stress_test/nb_active_scalability_multinet.py (hunk 2 above)
Comment (konstantinos-papadopoulos) [inline stress_test/nb_active_scalability_multinet.py]: Removed commented code (cleanup of multinet worker VMs)

File: emulators/multinet/build.sh (hunk 1 above)
Comment (konstantinos-papadopoulos) [inline emulators/multinet/build.sh]: Changed cloning to master branch of multinet

File: stress_test/nb_active_scalability_multinet.py (hunk 4 above)
Comment (konstantinos-papadopoulos) [inline stress_test/nb_active_scalability_multinet.py]:
```python
 t_start = time.time()
            result_get_sw = multinet_utils.multinet_command_runner(multinet_handlers_set.get_switches_handler,
                '[get_switches_handler]', multinet_base_dir,
                is_privileged=False)
            switches_measurement_latency_interval = time.time() - t_start
            logging.info('switches_measurement_latency_interval :{0} [sec]'.format(switches_measurement_latency_interval))
'''
We make a t_start point
We call the get switches handler for multinet
We calculate the switches_measurement_latency_interval (how long it took NSTAT) to get the number of all network switches
We display it with NSTAT logging message

We said we will keep this feature in NSTAT under the logging messages

Comment (anastop) [inline stress_test/nb_active_scalability_multinet.py]: yes, but why did we add the call for getting the switches? did we have this in the previous version of the NB test?

File: stress_test/sample_test_confs/beryllium/beryllium_nb_active_scalability_multinet.json (hunk 3 above)
Comment (konstantinos-papadopoulos) [inline stress_test/sample_test_confs/beryllium/beryllium_nb_active_scalability_multinet.json]: Parameters from master branch restored

File: .travis.yml [hunk 5]
```diff
@@ -1,6 +1,7 @@
 branches:
   only:
-    - master 
+    - master
+    - develop_measure_latency
```
Comment (anastop) [inline .travis.yml]: we do not need this anymore

**Pull request merged**